server/templates/topic_page.html

```diff
@@ -18,25 +18,29 @@
 {% set main_id = 'dc-topic-page' %}
 {% set title = topic_name + ' in ' + place_name + ' - ' + _('Topic Explorer') %}
 {% set subpage_title = _('Topic Explorer') %}
-{% set subpage_url = url_for('topic_page.topic_page') %}
+{# set subpage_url = url_for('topic_page.topic_page') #}
```

Comment:
curious what '#' does compared to '%' & why are we setting subpage_url twice?

Reply:
{# #} are comment blocks in jinja. it's meant to be temporary and we should revert once we have more topic pages and a decent landing page. i'll add a comment.